### DISCUSSION TITLE
Want to provide skilled support

### DISCUSSION THREAD

**Discussion by 733737** [OpenRCT2/OpenGraphics#32] [Ideas]
I am good at making ui, I am interested in making all of the menu icons and other assets, my main issue is not knowing the file structure or formats. Please help me, I'd be happy to make original art for the UI of OpenRCT2

**Comment by oli414** [comment]
The UI additions are currently not tracked on OpenGraphics, as it's mostly to extend the existing UI at this point.

Keep an eye on open pull requests on OpenRCT2 and you can offer suggestions for icons that you feel may be necessary. Just share your ideas and graphics on #open-graphics.

You can either help to implement the icon yourself, or ask the creator of the PR to include the graphics that you have made.
That's generally how I go about it, for example with the enhanced transparency options PR by IntelOrca:
https://github.com/OpenRCT2/OpenRCT2/pull/16797
